dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/JdbcSelectTask.java

```diff
@@ -44,13 +53,13 @@ public String getSql() {
   }
 
   @Override
-  protected Void doInConnection(
+  protected Summary doInConnection(
       @Nonnull TaskRunContext context,
       @Nonnull JdbcHandle jdbcHandle,
       @Nonnull ByteSink sink,
       @Nonnull Connection connection)
       throws SQLException {
-    ResultSetExtractor<Void> rse = newCsvResultSetExtractor(sink, -1);
+    ResultSetExtractor<Summary> rse = newCsvResultSetExtractor(sink, -1, interval);
     return doSelect(connection, rse, sql);
```

Comment:
If we have a `JdbcSelectIntervalTask` sublass, this would become `return doSelect(connection, rse, sql).withInterval(interval)`.